### PR TITLE
doc: matter: finalize structure for 2.1.0

### PR DESCRIPTION
**Pull request by greg-fer**
Finalized structure of Matter doc pages for NCS 2.1.0.
Renamed several pages.

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/ug_matter_overview_network_topologies.rst [hunk 1]
```diff
@@ -107,7 +107,11 @@ Matter network commissioning
 
 The commissioning in Matter is a process of joining a device to a fabric (operational network).
 The process takes place between a commissioner device and a commissionee device.
-It consists of the following stages:
+
+To start the commissioning procedure, the controller must get the onboarding information from the Matter accessory device.
+The data payload, which includes the device discriminator and setup PIN code, is encoded within a QR code, printed to the UART console, and can be shared using an NFC tag.
```
Comment (Damian-Nordic) [inline doc/nrf/ug_matter_overview_network_topologies.rst:112]:
```suggestion
The data payload, which includes the device discriminator and setup passcode, is encoded within a QR code, printed to the UART console, and can be shared using an NFC tag.
```
Updated the nomenclature.

File: doc/nrf/ug_matter_gs_testing_wifi_mobile.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+Configuring the environment
+***************************
+
+To configure and use CHIP Tool for Android, complete the following steps:
+
+1. Program the development kit for the Matter accessory device with one of available :ref:`matter_samples`.
+#. Configure the CHIP Tool for Android:
+
+   a. Install the controller using one of the options described in :ref:`ug_matter_configuring_controller_mobile`.
+   #. Complete the following steps from the :doc:`matter:nrfconnect_android_commissioning` user guide in the Matter documentation:
```
Comment (Damian-Nordic) [inline doc/nrf/ug_matter_gs_testing_wifi_mobile.rst:32]: Have we already updated `nrfconnect_android_commissioning.md` for Wi-Fi? If not, we may need another task to do so.

File: doc/nrf/ug_matter_gs_testing.rst [hunk 3]
```diff
@@ -8,11 +9,18 @@ When you build any of the available Matter samples to the supported development
...
+To control the Matter accessory device remotely over either of these networks, you need to set up a Matter controller on a mobile or a PC.
 Matter controller is a role within the Matter environment.
 This controller interacts with the accessory devices using the following protocols:
 
-* Bluetooth® LE during the commissioning process - to securely pass the network credentials and provision the accessory device into the Thread network.
+* Bluetooth® LE during the commissioning process - to securely pass the network credentials and provision the accessory device into the Thread network during commissioning.
+  At this stage, the controller has the commissioner role.
+
+  To start the commissioning procedure, the controller must get the onboarding information from the Matter accessory device.
+  The data payload, which includes the device discriminator and setup PIN code, is encoded within a QR code, printed to the UART console, and can be shared using an NFC tag.
```
Comment (Damian-Nordic) [inline doc/nrf/ug_matter_gs_testing.rst:20]:
```suggestion
  The data payload, which includes the device discriminator and setup passcode, is encoded within a QR code, printed to the UART console, and can be shared using an NFC tag.
```

**Review by Damian-Nordic — APPROVED** [review]

File: doc/nrf/ug_matter_gs_testing_wifi_mobile.rst (hunk 2 above)
Comment (greg-fer) [reply doc/nrf/ug_matter_gs_testing_wifi_mobile.rst:32]: We haven't, no.

File: doc/nrf/ug_matter_gs_testing.rst (hunk 3 above)
Comment (greg-fer) [reply doc/nrf/ug_matter_gs_testing.rst:20]: I've actually removed this line and kept it only in the Commissioning section.

**Review by LuDuda — APPROVED** [review]
Looks awesome! Thanks :+1:

**Review by kkasperczyk-no — APPROVED** [review]

**Pull request merged**